exporter/exporter.go

```diff
@@ -95,29 +107,59 @@ func (e *Exporter) Describe(ch chan<- *prometheus.Desc) {
 
 // Collect collects data to be consumed by prometheus
 func (e *Exporter) Collect(ch chan<- prometheus.Metric) {
-	data, err := e.client.GetDevices()
+	devices, err := e.client.GetDevices()
 	if err != nil {
 		log.Errorf("Fetching device stats failed: %v", err)
 		return
 	}
 
-	err = e.processMetrics(data, ch)
+	appliances := &types.GetAppliancesResult{}
+	for _, dev := range devices.Devices {
+		if dev.IsRemoElite() {
+			appliances, err = e.client.GetAppliances()
+			if err != nil {
+				log.Errorf("Fetching appliances stats failed: %v", err)
+				return
+			}
+			break
+		}
+	}
+
+	err = e.processMetrics(devices, appliances, ch)
 	if err != nil {
 		log.Errorf("Processing the metrics failed: %v", err)
 		return
 	}
 
 }
 
-func (e *Exporter) processMetrics(devicesResult *types.GetDevicesResult, ch chan<- prometheus.Metric) error {
+func (e *Exporter) processMetrics(devicesResult *types.GetDevicesResult, appliancesResult *types.GetAppliancesResult, ch chan<- prometheus.Metric) error {
 	for _, d := range devicesResult.Devices {
+		if !d.IsRemo() {
```

Comment:
I'm assuming this fixes #17 . WDYT @sorah ?

Reply:
Should fix,

Unrelated to #17 but I recently know that the Remo mini device only contains a temperature event, as it [equips a temperature sensor only](https://nature.global/faq/remo/1-12). Thus I don't think relying on device model names is a good idea.

Reply:
I see.
I will fix it referencing #17.